composer.json

```diff
@@ -5,7 +5,6 @@
     "keywords": ["templating"],
     "homepage": "https://twig.symfony.com",
     "license": "BSD-3-Clause",
-    "minimum-stability": "dev",
```

Comment:
using dev stability for our dependencies was done on purpose. You should use `phpunit/phpunit": "^11.4@stable"` to override it only for phpunit instead.

Reply:
Is it really needed? Looking at the dependencies:
https://github.com/twigphp/Twig/blob/c192cbff74c3f46c6dbb3c7b70cf72a4c475d571/composer.json#L26-L35

Reply:
Btw, the same thing happened when @VincentLanglet added PHPStan. It was installing the dev version without anybody noticing it.

Reply:
"phpstan/phpstan": "^2.0@stable" 

should be preferred too then